contracts/feature-tests/basic-features/src/sparse_array_features.rs

```diff
@@ -0,0 +1,51 @@
+elrond_wasm::imports!();
+
+#[elrond_wasm::module]
+pub trait SparseArrayFeatures {
+    #[endpoint]
+    fn sparse_array_create(&self, len: usize) -> SparseArray<Self::Api, 100> {
+        SparseArray::new(len)
+    }
+
+    #[endpoint]
+    fn sparse_array_get(&self, array: SparseArray<Self::Api, 100>, index: usize) -> usize {
+        array.get(index)
+    }
+
+    #[endpoint]
+    fn sparse_array_create_and_get(
+        &self,
+        len: usize,
+        index: usize,
+    ) -> MultiValue2<usize, SparseArray<Self::Api, 100>> {
+        let array = SparseArray::new(len);
+        let value = array.get(index);
+
+        (value, array).into()
+    }
+
+    #[endpoint]
+    fn sparse_array_set(
+        &self,
+        array: SparseArray<Self::Api, 100>,
+        index: usize,
+        value: usize,
+    ) -> SparseArray<Self::Api, 100> {
+        let mut array_clone = array;
+        array_clone.set(index, value);
```

Comment:
would rather name this `mut_array`, it is not a clone

Reply:
It was originally actually `array_clone = array.clone()`, but Clippy complained. Renamed now.